File: Ad-Hoc/pagina02/1267/1267.cpp
#include <iostream>
#include <vector>

using namespace std;

int main(){
    int num_ex_alunos, num_eventos;
    cin  >> num_ex_alunos >> num_eventos;

    if(num_ex_alunos == 0 && num_eventos == 0) break;

    vector<int> contagem_participacoes;
    for(int i = 0; i < num_ex_alunos; i++)
        contagem_participacoes.push_back(0);

    for(int i = 0; i < num_eventos; i++){
        vector<int> participacoes;
        for(int j = 0; j < num_ex_alunos; j++){
            int participacao;
            cin >> participacao;
            participacoes.push_back(participacao);
        }
        for(int j = 0; j < participacoes.size(); j++){
            contagem_participacoes[j] += participacoes[j];
        }
    }

    // ...continuar print("yes" if num_eventos in contagem_participacoes else "no")
}

File: Ad-Hoc/pagina02/1267/1267.py
while True:
    num_ex_alunos, num_eventos = [int(x) for x in input().split()]
    if num_ex_alunos == num_eventos == 0:
        break
    
    contagem_participacoes = [0 for i in range(num_ex_alunos)]
    for i in range(num_eventos):
        participacoes = [int(x) for x in input().split()]
        for idx, participacao in enumerate(participacoes):
            contagem_participacoes[idx] += participacao
            
    print("yes" if num_eventos in contagem_participacoes else "no")